frontend/src/component/admin/groups/EditGroup/EditGroup.tsx

```diff
@@ -13,10 +13,24 @@ import { useGroup } from 'hooks/api/getters/useGroup/useGroup';
 import { UG_SAVE_BTN_ID } from 'utils/testIds';
 import { GO_BACK } from 'constants/navigate';
 import { useGroups } from 'hooks/api/getters/useGroups/useGroups';
+import { IGroup } from 'interfaces/group';
 
-export const EditGroup = () => {
+export const EditGroupContainer = () => {
     const groupId = Number(useRequiredPathParam('groupId'));
-    const { group, refetchGroup } = useGroup(groupId);
+    const { group } = useGroup(groupId);
+
+    if (!group) return null;
+
+    return <EditGroup group={group} groupId={groupId} />;
+};
+
+interface IEditGroupProps {
+    group: IGroup;
+    groupId: number;
+}
+
+export const EditGroup = ({ group, groupId }: IEditGroupProps) => {
+    const { refetchGroup } = useGroup(groupId);
```

Comment:
I'm guessing in the end it doesn't matter much, but does it make sense to call the hook again for `refetchGroup` instead of passing it as a prop?

Reply:
True. I wanted to pass as little as possible, but I think your suggestion makes more sense after thinking about it. I'll make the change!